cli/command/service/rollback.go

```diff
@@ -0,0 +1,67 @@
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/versions"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func newRollbackCommand(dockerCli *command.DockerCli) *cobra.Command {
+	options := newServiceOptions()
+
+	cmd := &cobra.Command{
+		Use:     "rollback [OPTIONS]",
+		Aliases: []string{"rollback"},
```

Comment:
This alias is not necessary because it's the same as the first word in `Use:`, so this line can be removed.